### DISCUSSION TITLE
So far unable to get NUC8i7HVK to boot from Monterey or Ventura install media

### DISCUSSION THREAD

**Discussion by farmerpaul** [osy86/HaC-Mini#1065] [Q&A] [answered]
Hey folx,

I'm just getting started here, very excited about turning my NUC into a Mac!

I've been hammering away for the last several hours trying to get my NUC8i7HVK to recognize the bootable USB drive I prepared using `createinstallmedia` for either Monterey or Ventura, but so far no luck. This is after having updated the BIOS, resetting it and configuring it as described in the [BIOS settings step](https://osy.gitbook.io/hac-mini-guide/installation-guide/bios-settings).

For a while I was wondering if maybe my NUC was locked down or needed an NVRAM reset, so I opened it up, removed the jumper, and did what I think was reset the NVRAM, but still, the machine doesn't seem to think I've got a bootable USB attached. Tried all the USB ports, front and back.

I decided I should just test if my NUC is able to boot from _any_ bootable USB media, and sure enough, it has no problem booting from a bootable image of Linux Mint that I wrote to a USB stick. I also tried another USB drive that has the Windows 10 installer on it, and no problem there either, it booted fine.

So I'm now just wondering if I just missed something in the HaC-Mini instructions, and maybe I'm not actually supposed to be able to boot directly from a Monterey or Ventura installer disk, but instead should start at Big Sur and upgrade from there? This is just my next wild guess.

So I'm now downloading Big Sur now, which apparently will take all night thanks to Apple's remarkably limited CDN server speeds.

Any tips, tricks, or suggestions welcome!

**Comment by farmerpaul** [comment]
So, I made _some_ progress with Monterey/Ventura, but only a bit (Big Sur is still downloading).

Despite that the [install instructions](https://osy.gitbook.io/hac-mini-guide/installation-guide/installation#patching-osx-installer) say quite clearly to skip the "Patching OSX Installer" section if you're trying to install Big Sur and up (which I am), it seems that you do indeed have to run the `HaCMini.pkg` installer app on the install media, because that's the very thing that makes it possible for the NUC to recognize the installer as bootable media. I discovered that when I did NOT skip the entire "Patching OSX Installer" section, the NUC finally saw my bootable USB drive!

I'd thought that because I was installing Monterey or Ventura that I was only expected to run `HaCMini.pkg` _after_ I'd installed macOS, based on my reading and re-reading of the instructions. So I'm a bit confounded by the wording and placement of this notice:

<img width="934" alt="Screenshot 2023-03-09 at 1 17 21 AM" src="https://user-images.githubusercontent.com/1282772/223928745-cc61f35b-e9a1-4f95-b13e-059e9bc6cbb1.png">

It seems that what should be written there is, "Skip **Step 6** if this applies to you" (and it shouldn't precede the entire 7-step procedure), otherwise it sounds like I'm supposed to skip the entire "Patching OSX Installer" section. And maybe that section should not be titled "Patching OSX Installer", but rather something like "Install OpenCore loader onto OSX installer media".

Now, back to my challenges…

While the NUC indeed now recognizes that I have a bootable USB, and in the case of Ventura, it saw a partition called "OpenCore", when I tried to boot from that partition, it seemed to try and the screen flashed a bit maybe, but then it just booted up back into my old Windows 10 installation (which is what my NUC came with preinstalled). It also showed a couple other bootable partitions on the installer media, called things like "…Part 2…" and "…Part 3…", but trying to boot into either of those seems to do absolutely nothing.

In the case of Monterey, an "OpenCore" partition isn't even showing up at all – only the "…Part 2…" and "…Part 3…" partitions. And those ones similarly do nothing when I try to boot from them.

So… back to the drawing board. I'll give Big Sur a go when it's finished downloading.

**Comment by farmerpaul** [comment, marked as answer]
Whew… well, since I managed to get the USB media to be recognized as bootable, but no further, I since discovered there have been other folks in the same predicament, and in particular one very helpful solution that allowed me to finally boot into the installer: https://github.com/osy/HaC-Mini/discussions/955#discussioncomment-1603057

It'd be great to have these things better documented to save others some time down the road.

**Reply by osy** [reply]
Thanks for the feedback, I've updated the guide with your experience: https://github.com/osy/HaC-Mini/commit/1e02ea80fe0190d70de2706cd12fd53edf04b488

**Reply by farmerpaul** [reply]
Thanks @osy, appreciate that change.

Just FYI - it's still the case that I cannot start the Ventura installer without manually going into the UEFI shell, navigating to the folder containing OpenCore.efi on the EFI partition, and running it from there. So those 3 bullets you added in the docs still wouldn't have helped me make any progress, I'm afraid. Only having seen [this post](https://github.com/osy/HaC-Mini/discussions/955#discussioncomment-1603057) from @ulm0 helped me move forward. This behaviour is reliable on my system.

**Reply by osy** [reply]
You reset your BIOS and looked for the USB entry in F10 menu?

**Reply by farmerpaul** [reply]
Yup, innumerable times, banging my head against the wall, hoping I'd missed something haha. No luck. It's fine, I'm getting comfortable in the UEFI shell, definitely a handy tool!

**Reply by osy** [reply]
You said

> While the NUC indeed now recognizes that I have a bootable USB, and in the case of Ventura, it saw a partition called "OpenCore", when I tried to boot from that partition, it seemed to try and the screen flashed a bit

That happens if you had an existing OC install on your NVMe that is broken somehow. OC will set the NVRAM variable and hide any additional OC entries on other drives (like the USB). Resetting BIOS is the easiest way to clear NVRAM (the other ways involve loading OC or macOS).

If you don't have boot menu set on the USB OC install, then the USB OC could have tried to chain load the NVMe OC which would do the same thing as above.

If you either 1) wipe the NVMe OC + reset NVRAM or 2) reset BIOS + F10 menu + select name of your USB drive (if you see the text OpenCore in F10 menu at all, the NVRAM wipe failed) and 3) make sure boot menu is always displayed in the USB OC install, then it should work without the UEFI shell.

I think your experience is complicated by the fact that there's an existing but broken OC install and the guide assumes a clean NVMe. I think adding UEFI shell instructions would be confusing to newcomers.

**Reply by farmerpaul** [reply]
Ah. Sorry, a lot of this language is still foreign to me… like NVMe. I just Googled NVMe and I see it's kind of like an SSD technology, maybe? Gosh, this is making me feel old haha.

But because you say resetting BIOS is supposed to reset things like NVRAM, I'm wondering I just don't know how to reset the BIOS properly? I've just been assuming I enter the BIOS by hitting F2, then hitting F9 to reset the BIOS to default values, which I assume just resets the 3 settings mentioned in the first step of the docs. And then I just turn those settings back to what the docs say they're supposed to be at.

Anyway, I agree, don't change the docs based on my being an ignorant front-end software engineer, I'm sure if I had some background in hardware engineering none of this would seem nearly as scary!
